packages/client/src/App.tsx

```diff
@@ -1,63 +1,89 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client'
-import AccountView from '@components/AccountView'
 import MapSlideUpSheet from '@components/MapSlideUpSheet'
 import MapView from '@components/MapView'
-import { Business, BusinessType, DisplayableBusiness, Location } from '@futureproof/typings'
+import {
+  Business,
+  BusinessType,
+  DisplayableBusiness,
+  Location,
+} from '@futureproof/typings'
 import { NavigationContainer } from '@react-navigation/native'
 import { createStackNavigator, StackScreenProps } from '@react-navigation/stack'
 import Constants from 'expo-constants'
 import React from 'react'
-import { SafeAreaView, StyleSheet, TouchableOpacity, View } from 'react-native'
-import AccountButton from './components/AccountButton'
+import {
+  SafeAreaView,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  Dimensions,
+} from 'react-native'
+import AccountButton from '@components/AccountViews/AccountButton'
+import AccountView from '@components/AccountViews/AccountView'
+import HelpView from '@components/AccountViews/HelpView'
+import PasswordView from '@components/AccountViews/PasswordView'
+import PPView from '@components/AccountViews/PPView'
+import ToSView from '@components/AccountViews/ToSView'
+import BusinessView from '@components/BusinessViews/BusinessView'
 
-const ExampleBusiness : DisplayableBusiness =  {
+const ExampleBusiness: DisplayableBusiness = {
   id: '1',
   name: 'Starbucks',
-  profileText: 'This is a test business and there is not that much to say about it.',
+  profileText:
+    'This is a test business and there is not that much to say about it.',
   sustainabilityScore: 80,
   customerScore: 65,
   type: BusinessType.Cafe,
-  profilePicture: 'https://upload.wikimedia.org/wikipedia/en/thumb/d/d3/Starbucks_Corporation_Logo_2011.svg/1200px-Starbucks_Corporation_Logo_2011.svg.png'
+  profilePicture:
+    'https://upload.wikimedia.org/wikipedia/en/thumb/d/d3/Starbucks_Corporation_Logo_2011.svg/1200px-Starbucks_Corporation_Logo_2011.svg.png',
 }
 
 interface LocationType extends Pick<Location, 'latitude'> {
-  business : Pick<Business, 'sustainabilityScore'>
+  business: Pick<Business, 'sustainabilityScore'>
 }
 
 // Initialise Apollo Client
 const client = new ApolloClient({
   uri: `${Constants.manifest?.extra?.serverAddress}/graphql`, // Server URL (must be absolute)
-  cache: new InMemoryCache() // Cache
+  cache: new InMemoryCache(), // Cache
 })
 
+const width = Dimensions.get('window').width
+
 type RootStackParamList = {
-  MapView : undefined
```

Comment:
These types should not be undefined

Reply:
The route doesn't have any parameters so the documentation specifies to use undefined. Check it out at https://reactnavigation.org/docs/typescript/